README.md

```diff
@@ -82,9 +82,11 @@ dataset_ids = get_experiment_list_from_gene("Pvalb", axis='sagittal')
 - One can download any dataset from a dataset ID:
 ```python
 from atldld.sync import download_dataset_parallel
-dataset = download_dataset(dataset_id=DATASET_ID, **kwargs)
+dataset = download_dataset_parallel(dataset_id=DATASET_ID, **kwargs)
```

Comment:
Suggestion: rename `download_dataset_parallel` to `download_parallel_dataset`.

Why? The former sounds like it's a multi-threaded dataset download or something (unless you're Yoda 😜) Same for `get_transform_parallel`.

What do you think?

Reply:
hehe:) I agree! I will rename it:)

Reply:
Done in fd2a84061a80dc976ba03b17631ccff0ef88e0f8